test/CMakeLists.txt

```diff
@@ -849,14 +893,6 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_viirs_aod
                           -n 0.0"
                           viirs_aod.nc ${IODA_CONV_COMP_TOL_ZERO})
 
-ecbuild_add_test( TARGET  test_${PROJECT_NAME}_satbias_viirs_aod
```

Comment:
why are this one and the next one removed?

Reply:
yeah I agree this is getting to be a large file

Reply:
I wrote a python script that helped me find tests that were dropped by accident and added them back in.

```python
import re

path1 = "/Users/rmclaren/Work/ioda-bundle/iodaconv/test/CMakeLists.txt"
path2 = "/Users/rmclaren/Temp/develop_CMakeLists.txt"

dict1 = {}
dict2 = {}

expr = re.compile(".*TARGET\s+(?P<testFile>[^\s]+)")

with open(path1) as f:
    for line in f.readlines():
        if expr.match(line):
            m = expr.search( line )
            dict1[m.group('testFile')] = ''
            
with open(path2) as f:
    for line in f.readlines():
        if expr.match(line):
            m = expr.search( line )
            dict2[m.group('testFile')] = ''

print("Tests added.")
for key in dict1.keys():
    if not key in dict2.keys():
        print(key)

print("Missing Tests.")
for key in dict2.keys():
    if not key in dict1.keys():
        print(key)
```